WalletWasabi.Gui/Controls/WalletExplorer/SendControlViewModel.cs

```diff
@@ -268,6 +271,20 @@ protected SendControlViewModel(WalletService walletService, string title)
 						return;
 					}
 
+					BitcoinAddress customChangeAddress = null;
+					if (!string.IsNullOrWhiteSpace(CustomChangeAddress))
+					{
+						try
+						{
+							customChangeAddress = BitcoinAddress.Create(CustomChangeAddress.Trim(), Global.Network);
```

Comment:
If the Address and the Custom change is the same, an error should be thrown.

Reply:
We aren't doing that with the CustomFee either.

Reply:
Done.